src/test/java/org/humancellatlas/ingest/core/service/MetadataUpdateServiceTest.java

```diff
@@ -70,18 +65,12 @@ public void testUpdateShouldSaveAndReturnUpdatedMetadata() {
     @Test
     public void testUpdateShouldSetStateToDraftWhenContentChanged() {
         //given:
-        ObjectMapper mapper = new ObjectMapper();
-
-        ObjectNode content = mapper.createObjectNode();
-        ObjectNode projectCore0 = content.putObject("project_core");
-        projectCore0.put("project_title", "Old Project Title");
-
+        ObjectNode content = createProjectMetadataWithTitle("Old Project Title");
         Project project = new Project(content);
 
+        ObjectNode newContent = createProjectMetadataWithTitle("New Project Title");
         ObjectNode patch = mapper.createObjectNode();
-        ObjectNode newContent = patch.putObject("content");
-        ObjectNode projectCore = newContent.putObject("project_core");
-        projectCore.put("project_title", "New Project Title");
+        patch.set("content", newContent);
```

Comment:
Creation of the patch object is duplicated in all test. I would extract the common bits to a function so that each test case becomes clearer.